src/app/components/PortraitVideoCarousel/index.tsx

```diff
@@ -5,12 +5,36 @@ import Heading from '../Heading';
 import { LeftChevron, RightChevron } from '../icons';
 import styles, { PROMO_ITEM_WIDTH } from './index.styles';
 import { ServiceContext } from '../../contexts/ServiceContext';
+import PortraitVideoModal from '../PortraitVideoModal';
 
-interface PortraitVideoItem {
+export interface PortraitVideoItem {
```

Comment:
TICKET OF WORK:

I've enriched this interface (along with the `modal` interface) to capture ALL the parameters that are returned via the BFF. Continuing the top heavy approach, we can clean this up after the spike if we want so that we have a leaner production-ready response in the BFF and in Simorgh. That way, we're minimising on parameter pollution. Note: cleanup possibly required in both BFF and Simorgh depending on what parameters can be dropped. We should investigate that further too.